File: 351001/Ushakov/lab5-6/publisher/internal/mapper/creator.go
package mapper

import (
	creator "github.com/Khmelov/Distcomp/351001/Ushakov/labX/publisher/internal/model"
	"github.com/Khmelov/Distcomp/351001/Ushakov/labX/publisher/internal/storage/model"
)

func MapCreatorToModel(cr creator.Creator) model.Creator {
	return model.Creator{
		ID:        cr.ID,
		Login:     cr.Login,
		Password:  cr.Password,
		FirstName: cr.FirstName,
		LastName:  cr.LastName,
	}
}

func MapModelToCreator(cr model.Creator) creator.Creator {
	return creator.Creator{
		ID:        cr.ID,
		Login:     cr.Login,
		Password:  cr.Password,
		FirstName: cr.FirstName,
		LastName:  cr.LastName,
	}
}

File: 351003/Mashevskiy/Lab1/src/main/java/by/tracker/rest_api/model/Creator.java
package by.tracker.rest_api.model;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Creator {
    private Long id;
    private String login;
    private String password;
    private String firstName;
    private String lastName;
}